.github/workflows/cancel.yml

```diff
@@ -23,4 +25,8 @@ jobs:
       uses: peter-evans/close-issue@v1
       with:
         issue-number: ${{ steps.env_info.outputs.issuenm }}
-        comment: Cancelling rollout as requested by ${{ steps.env_info.outputs.requestor }}.
\ No newline at end of file
+        comment: Cancelling rollout as requested by @${{ steps.env_info.outputs.requestor }}.
+    - name: Notify chat
+      run: |
+       echo "Thread key GIIHUB_REPOSITORY-${{ steps.env_info.outputs.issuenm }} requestor ${{ steps.env_info.outputs.requestor }}"
+        [ ! -z "$URL" ] && curl "$URL&threadKey=$GIIHUB_REPOSITORY-${{ steps.env_info.outputs.issuenm}}" -H "Content-Type: application/json" -d '{ "text" : "\uD83E\uDDEF cancelling rollout as requested by ${{ steps.env_info.outputs.requestor }}" }'
```

Comment:
`GIIHUB` here looks like a typo (same in the other workflow files) but otherwise looks good and the idea sounds really useful 👍

Reply:
nice catch - didn't effect the functionality atm but was unintended and could effect future threading. I'd like to blame this butterfly keyboard but that doesn't account for the missing `T`